numba/tests/test_fancy_indexing.py

```diff
@@ -406,7 +406,7 @@ def numba_get_item(array, idx):
         expected = get_item(arr, index)
         got = numba_get_item(arr, index)
         # Sanity check: In advanced indexing, the result is always a copy.
-        self.assertNotIn(expected.base, orig_base)
+        self.assertIsNot(expected.base, orig_base)
```

Comment:
What prompted this change?

Reply:
This triggered an error because other changes made the shape comparison logic different. Graham and I looked at the original PR, it was originally an is-not comparison though not using the unit test assertion, the review comment was ambiguous in that is said to use the method "in" the unittest system, and had it changed to not-in, which isn't the point of this comparison.

Reply:
Thanks for the explanation.